### DISCUSSION TITLE
NativeMessagingHosts entry in Windows registry switches to wrong path

### DISCUSSION THREAD

**Discussion by WernerMue** [keepassxreboot/keepassxc#11166] [Q&A] [answered]
KeepassXC 2.7.8 r2 enterprise installation changes registry setting for NativeMessagingHosts always to C:/Program Files/KeePassXC/org.keepassxc.keepassxc_browser_edge.json despite missing write permissions in that folder, It does not change to C:/Users/xxxxxx/AppData/Local/KeePassXC/org.keepassxc.keepassxc_browser_edge.json as expected. Also manually setting the correct destination results in same after restart.

Any idea what is wrong?

Thanks

**Comment by droidmonkey** [comment, marked as answer]
Update to 2.7.9 or stop using the portable version in an installed location. 

https://github.com/keepassxreboot/keepassxc/pull/10760

**Reply by WernerMue** [reply]
this is working, thanks!
